### PR TITLE
change amounts to bytes

### PR DESCRIPTION
**Pull request by vedhavyas**

### CODE REVIEW

File: purchaseorder/purchaseorder.proto [hunk 1]
```diff
@@ -31,11 +31,11 @@ message PurchaseOrderData {
   // ISO currency code 
   string currency = 12;
   // ordering gross amount including tax 
-  int64 order_amount = 13;
+  bytes order_amount = 13;
```
Comment (mikiquantum) [inline purchaseorder/purchaseorder.proto]: out of curiosity, what is the difference between `order-amount` and `gross_amount` ?

File: invoice/invoice.proto [hunk 2]
```diff
@@ -33,11 +33,11 @@ message InvoiceData {
...
   // invoice amount including tax 
-  int64 gross_amount = 14;
+  bytes gross_amount = 14;
   // invoice amount excluding tax 
-  int64 net_amount = 15;
-  int64 tax_amount = 16;
-  int64 tax_rate = 17;
+  bytes net_amount = 15;
+  bytes tax_amount = 16;
+  bytes tax_rate = 17;
```
Comment (mikiquantum) [inline invoice/invoice.proto]: do we treat rates (%) same as decimals?

Comment (vedhavyas) [reply invoice/invoice.proto]: yeah since percentages can have decimals, no ?

File: purchaseorder/purchaseorder.proto (hunk 1 above)
Comment (vedhavyas) [reply purchaseorder/purchaseorder.proto]: Not sure of the distinction. Assumed its PO specific nomenclature

**Pull request merged**